Units/parser-eiffel.r/obsolete.e.d/input.e

```diff
@@ -1,4 +1,4 @@
-class OBSOLETE obsolete "Use a different one"
+class OBSOLETE_ obsolete "Use a different one"
```

Comment:
Eiffel is actually case-insensitive so a class name matching a keyword is a syntax error.

As indexing.e.d has the same problem I assume it was a misunderstanding of the original author.  Assuming this I interpreted the test as simply attempting to ensure that the keyword "obsolete" doesn't break the parser.  Perhaps it did in the past?